src/lib.rs

```diff
@@ -1,9 +1,36 @@
 //! A library to interact with the Windows API for display settings.
 //!
 //! This library provides an abstraction around some `winuser.h` calls relevant for modifying display settings.
+#![deny(
```

Comment:
I would rather leave this as default and have a check in the CI (GitHub actions), but I guess this is my personal preference

Reply:
OK, it's a standard practice I put in my code - but I'll remove it for now. Fixed in 018306c.